openhands/server/routes/manage_conversations.py

```diff
@@ -1,3 +1,11 @@
+# IMPORTANT: LEGACY V0 CODE
```

Comment:
This file seems a bit tricky since its handling both v0 and v1 conversations? (e.g `search_conversations` endpoint)